src/index.js

```diff
@@ -37,7 +37,12 @@ function buildValidations(referenced, dereferenced, receivedOptions) {
     const schemas = {};
 
     const basePaths = dereferenced.servers && dereferenced.servers.length
```

Comment:
i actually think that it would be best too extract this whole logic to its own function if you don't mind